cfme/common/cloudinfra_provider.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+from .provider import base_types
+
+# there are a lot of tests which request any of Cloud or Infra providers
+# CloudInfraProvider has been left in order not to
+# replace all such requests with both base Cloud and Infra providers
+CloudInfraProvider = [val for t, val in base_types().items() if t in ('infra', 'cloud')]  # noqa
```

Comment:
cant this just be the mixin as aslias?

Reply:
I just replaced all CloudInfra calls with CloudProvider and InfraProvider.